easybuild/easyconfigs/n/netcdf4-python/netcdf-python-1.4.1-CrayGNU-18.08-python3.eb

```diff
@@ -0,0 +1,33 @@
+#
+name = 'netcdf-python'
+version = '1.4.1'
+
+homepage = 'http://unidata.github.io/netcdf4-python/'
+description = """Python/numpy interface to netCDF."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+toolchainopts = {'verbose': False}
+
+source_urls = ['https://pypi.python.org/packages/source/n/netCDF4/']
+sources = ['netCDF4-%(version)s.tar.gz']
+
+req_py_majver = 3
+req_py_minver = 6
+
+pythonver = '%s.%s' % (req_py_majver, req_py_minver)
+versionsuffix = '-python%s' % req_py_majver
+
+dependencies = [
+    ('cray-python/3.6.1.1', EXTERNAL_MODULE),
```

Comment:
There is a newer version:
('cray-python/3.6.5.1', EXTERNAL_MODULE),

Reply:
I can change that.